textworld/generator/__init__.py

```diff
@@ -175,14 +176,34 @@ def make_game(world_size: int, nb_objects: int, quest_length: int,
     world = make_world(world_size, nb_objects=0, rngs=rngs)
 
     # Sample a quest according to quest_length.
-    options = ChainingOptions()
+    class Options(ChainingOptions):
+
+        def get_rules(self, depth):
+            if depth == 0:
+		        # Last action should not be "go <dir>".
+                return data.get_rules().get_matching("^(?!go.*).*")
```

Comment:
You can just do `options.rules_per_depth = [data.get_rules().get_matching(...)]`

Reply:
Oh right. I think I had this line before you added the `rules_per_depth` attribute.

Reply:
This is implemented in #44.